### PR TITLE
CN-258 adding primary keys

### PR DESCRIPTION
**Pull request by tc-dan-bosler**
Added Primary Keys to the sales_order_item_purchase_order & quote_item_purchase_order tables via the etc/db_schema.xml file.

### CODE REVIEW

**Comment by tc-dan-bosler** [comment]
@tc-greg-lee once I can get Jay to approve the review, could you approve this merge to the main branch?  Thanks

**Review by tc-greg-lee — APPROVED** [review]
Thanks!

File: etc/db_schema.xml [hunk 1]
```diff
@@ -10,6 +10,9 @@
         <column xsi:type="varchar" name="po_number" nullable="false" default="" length="255" comment="PO Number"/>
         <column xsi:type="varchar" name="order_request_id" nullable="false" default="" length="255" comment="Punchout Request Id"/>
         <column xsi:type="text" name="extra_data" nullable="true" comment="Punchout Extra Data"/>
+		<constraint xsi:type="primary" referenceId="PRIMARY">
```
Comment (p2g-jay-grayson) [inline etc/db_schema.xml:13]: Is this just GH not formatting it correctly?  Or is not aligned with the other nodes?

File: etc/db_schema.xml [hunk 2]
```diff
@@ -22,6 +25,9 @@
...
```
Comment (p2g-jay-grayson) [inline etc/db_schema.xml:28]: Is this just GH not formatting it correctly?  Or is not aligned with the other nodes?

**Review by p2g-jay-grayson — CHANGES_REQUESTED** [review]
See inline

**Review by tc-dan-bosler — COMMENTED** [review]
Thanks! it should be aligned now!

**Review by p2g-jay-grayson — APPROVED** [review]

**Pull request merged**